pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -909,7 +913,7 @@ type ServiceSpec struct {
 // +k8s:openapi-gen=true
 type TiDBServiceSpec struct {
 	// +k8s:openapi-gen=false
```

Comment:
Should this be changed to `true`?

Reply:
can be changed in another PR? This does not relate to crd.

Reply:
Yes, it's not related to CRD generation, however, with `false`, the following fields will not be shown in the final CRD files, what about we just fix this in this PR?

Reply:
All fields in `ServiceSpec` have been generated in current CRD.

This comment only affect openapi spec generation.